pay-api/src/pay_api/services/eft_service.py

```diff
@@ -162,15 +162,23 @@ def create_shortname_refund(request: Dict[str, str], **kwargs) -> Dict[str, str]
 
         current_app.logger.debug(f'Starting shortname refund : {shortname_id}')
 
-        refund = EftService._create_refund_model(request, shortname_id, amount, comment)
+        refund = EftService._create_refund_model(request, shortname_id, amount, comment).flush()
         EftService._refund_eft_credits(int(shortname_id), amount)
 
+        EFTHistoryService.create_shortname_refund(
```

Comment:
When implementing approve/decline we need to find this history record by eft_refund_id and update the status.

For decline, we will need to return the refund amount back to the short name as well